Error during install: cannot find module 'bluebird'

I'm running into the following error when trying to install that plugin:

```
$ heroku plugins:install heroku-ci
Installing plugin heroku-ci...module.js:442
    throw err;
    ^

Error: Cannot find module 'bluebird'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/steve/.local/share/heroku/plugins/node_modules/heroku-ci/lib/git.js:2:17)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
 !
 ▸    Error installing plugin heroku-ci
 !    error installing plugin heroku-ci
```

Any idea of what I'm doing wrong here?

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
   },
   "dependencies": {
     "ansi-escapes": "1.4.0",
+    "bluebird": "^3.4.6",
     "co": "^4.6.0",
     "co-wait": "0.0.0",
     "github-url-to-object": "^2.2.6",

